### PR TITLE
feat(react-components): page controls

### PR DESCRIPTION
**Pull request by jeysal**

### CODE REVIEW

File: packages/react-components/src/icons/PreviousPage_Inactive.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fampinheiro) [inline packages/react-components/src/icons/PreviousPage_Inactive.svg:1]: Is it intentional to keep this as an svg?

Comment (jeysal) [reply packages/react-components/src/icons/PreviousPage_Inactive.svg:1]: oops, will delete

File: babel.config.js [hunk 2]
```diff
@@ -5,7 +5,11 @@ module.exports = {
   ],
   plugins: [
     [require.resolve('@babel/plugin-transform-runtime'), { corejs: 3 }],
-    [require.resolve('babel-plugin-transform-inline-environment-variables')],
+
+    require.resolve('babel-plugin-transform-inline-environment-variables'),
+
+    require.resolve('babel-plugin-lodash'),
```
Comment (kmaid) [inline babel.config.js:11]: Is this still required?

Comment (jeysal) [reply babel.config.js:11]: I included it because why not, even the Serverless deployment doesn't need to be bigger than it has to :sweat:

**Review by kmaid — APPROVED** [review]

**Comment by jeysal** [comment]
restarting Jest in hopes that codecov check will update

**Comment by jeysal** [comment]
yaaaay it did

**Comment by jeysal** [comment]
although it has no clue about diff with master
but whatever, the component has a trillion branches and 100% of it (at least the component itself) is covered, so this is definitely an overall increase

**Pull request merged**